Lesson_02/hometask/l02_t02_count_vowels.py

```diff
@@ -1,6 +1,8 @@
+# Homework: Lesson 2 [Volkov Anton]
```

Comment:
All this better to format as multiline docstring smth aka
"""
Lesson2 Hometask.

Task description...
"""

This will solve linter error in addition 
./T-HOMETASKS/L02/l02_h01_AntonVolkov.py:1:1: D100 Missing docstring in public module